code/modules/antagonists/ert/ert.dm

```diff
@@ -190,3 +194,17 @@
 
 	missiondesc += "<BR><B>Your Mission</B> : [ert_team.mission.explanation_text]"
 	to_chat(owner,missiondesc)
+
+/datum/antagonist/ert/clown/greet()
+	if(!ert_team)
+		return
+
+	to_chat(owner, "<B><font size=3 color=red>You are the [name].</font></B>")
+
+	var/missiondesc = "Your squad is being sent on a mission to [station_name()] by Nanotrasen's Comedy Division."
+	if(leader) //If Squad Leader
+		missiondesc += " You are the worst clown here. As such, you were able to stop slipping the admiral for long enough to be given command. Good luck, honk!"
+	else
+		missiondesc += " Follow orders given to you by your squad leader in order to ensure maximum laughs."
```

Comment:
```suggestion
		missiondesc += " Follow orders given to you by your squad leader to ensure maximum laughs."
```

Reply:
it is in order.

your statement would make it be "leader to ensure maximum laughs" suggesting that the leader is leading to ensure maximum laughs. in order makes it so by the action of following orders, you would me ensuring maximum laughs.